app/services/preserver/blind_importer.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+class Preserver::BlindImporter
+  def self.import(id:, change_set_persister:, importing_metadata_adapter: default_importing_metadata_adapter)
+    new(id: id, change_set_persister: change_set_persister, importing_metadata_adapter: importing_metadata_adapter).import!
+  end
+
+  def self.default_importing_metadata_adapter
+    FileMetadataAdapter.new(storage_adapter: importing_storage_adapter)
+  end
+
+  def self.importing_storage_adapter
+    Valkyrie::StorageAdapter.find(:google_cloud_storage)
+  end
+
+  attr_reader :id, :importing_metadata_adapter, :change_set_persister
+  delegate :storage_adapter, to: :change_set_persister
+  def initialize(id:, importing_metadata_adapter:, change_set_persister:)
+    @id = id
+    @importing_metadata_adapter = importing_metadata_adapter
+    @change_set_persister = change_set_persister
+  end
+
+  def import!
+    import_file_identifiers
+    member_ids = importing_resource.try(:member_ids) || []
+    member_ids.map! do |member_id|
+      begin
+        member = self.class.import(id: member_id, importing_metadata_adapter: importing_metadata_adapter.with_context(parent: importing_resource), change_set_persister: change_set_persister)
+        importing_change_set.created_file_sets += [member] if member.is_a?(FileSet)
+        member.id
+      rescue Valkyrie::Persistence::ObjectNotFoundError
+        nil
+      end
+    end.compact!
+    output = change_set_persister.save(change_set: importing_change_set)
+    output
+  end
+
+  def import_file_identifiers
```

Comment:
looks like we're importing the actual files as part of this method,  not just getting the ids. rename?